.env.ci

```
@@ -1,6 +1,11 @@
+# AWSConfig
 APP_AWS_ACCESS_KEY_ID=dummy
 APP_AWS_SECRET_ACCESS_KEY=dummy
 
+## DynamoDBConfig
 APP_AWS_DYNAMODB_REGION=ap-northeast-1
 APP_AWS_DYNAMODB_ENDPOINT_URL=http://localhost:7999# CI環境では7999ポート
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_AWS_DYNAMODB_ENDPOINT_URL key should go before the APP_AWS_DYNAMODB_REGION key

(UnorderedKey)